garbage-classifier/result.txt

```diff
@@ -0,0 +1,3 @@
+Single process prediction time: 53.85 seconds
+8-process prediction time: 48.23 seconds
+Speedup: 1.12 times
```

Comment:
interesting, looks like model is using multi cpu under the good even with one process